test/k8sT/Services.go

```diff
@@ -779,7 +779,7 @@ var _ = Describe("K8sServicesTest", func() {
 			// packets.
 
 			// Field #7 is "RxPackets=<n>"
-			cmdIn := "cilium bpf ct list global | awk '/%s/ { sub(\".*=\",\"\", $7); print $7 }'"
+			cmdIn := "cilium bpf ct list global | perl -ne 'print $1 if /%s.*RxPackets=(\\d+).*/'"
```

Comment:
These changes appear to be made in the commit which introduces a new `--time-diff` flag, but these commands aren't using the new flag. Is the default behaviour changing? Should this change be in a different commit?

Also, I would be reluctant to start adding `perl` usage into the codebase simply to minimize the number of different languages we use. I'm happy to be overruled if we think perl knowledge is more common across the Cilium contributor community than awk (they're both a bit foreign to me), but I'll at least raise it as a point here since this seems like this would be the first perl expression.

Reply:
+1

Reply:
> These changes appear to be made in the commit which introduces a new --time-diff flag, but these commands aren't using the new flag. Is the default behaviour changing? Should this change be in a different commit?

Makes sense.

> Also, I would be reluctant to start adding perl usage into the codebase simply to minimize the number of different languages we use. I'm happy to be overruled if we think perl knowledge is more common across the Cilium contributor community than awk (they're both a bit foreign to me), but I'll at least raise it as a point here since this seems like this would be the first perl expression.

I thnk the perl one liner above is more easy to understand than the awk above because it is just a single regular expression. Even if you don't understand all the details, I think you get an idea of what it does. That's my personal opinion colored by my personal bias though :-)

Reply:
Change removed.